src/test/java/uk/gov/companieshouse/certificates/orders/api/validator/OptionsValidatorHelperTest.java

```diff
@@ -40,4 +43,119 @@ void correctlyErrorsWhenCompanyTypeIsNull() {
         assertThat(result, is(false));
         assertThat(errors, containsInAnyOrder("company type: is a mandatory field"));
     }
+
+    @Test
+    void correctlyErrorsWhenActiveLimitedCompanyAndLiquidatorsDetailsSupplied() {
+        // Given
+        final CertificateItemOptions options = new CertificateItemOptions();
+        LiquidatorsDetails liquidatorsDetails = new LiquidatorsDetails();
+        options.setLiquidatorsDetails(liquidatorsDetails);
+        OptionsValidationHelper helper =
```

Comment:
Use mock object

Reply:
Mock object used for RequestValidatable, but real object used for CertificateItemOptions